modules/installation-guide/partials/proc_installing-che-on-codeready-containers-using-chectl.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-codeready-containers
+
+[id="installing-{prod-id-short}-on-codeready-containers-using-{prod-cli}_{context}"]
+= Installing {prod-short} on CodeReady Containers using {prod-cli}
+
+This section describes how to install {prod-short} on CodeReady Containers using {prod-cli}.
+
+.Prerequisites
+
+* The `{prod-cli}` management tool is available. See xref:using-the-chectl-management-tool.adoc[].
+* A running instance of CodeReady Containers. See xref:using-codeready-containers-to-set-up-openshift-4_{context}[].
+
+.Procedure
+
+* Run the following command:
```

Comment:
Empty information - I usually try to say what the command actually does, and perhaps explicate what the example does, if there are variables.